File: assignment_1/Owner.java
// This class is for the person who owns the store.
public class Owner {
    int uid;
    int o_id;
    String User_name;
    String contact;
    String email;
    String address;

    // Prints all owner details
    void displayOwner() {
        System.out.println("Owner ID : " + o_id);
        System.out.println("User ID  : " + uid);
        System.out.println("Name     : " + User_name);
        System.out.println("Contact  : " + contact);
        System.out.println("Email    : " + email);
        System.out.println("Address  : " + address);
    }

    // Shows that owner is managing the store
    void ManageStore() {
        System.out.println(User_name + " is managing the store.");
    }
}

File: python/owner.py
class Owner:
    def __init__(self):
        self.uid = 0
        self.o_id = 0
        self.User_name = ""
        self.contact = ""
        self.email = ""
        self.address = ""

    def displayOwner(self):
        print("Owner ID :", self.o_id)
        print("User ID  :", self.uid)
        print("Name     :", self.User_name)
        print("Contact  :", self.contact)
        print("Email    :", self.email)
        print("Address  :", self.address)

    def ManageStore(self):
        print(self.User_name, "is managing the store.")